extension/src/setup/autoInstall.ts

```diff
@@ -0,0 +1,64 @@
+import { getPythonExecutionDetails } from '../extensions/python'
```

Comment:
When I install DVC but the project isn't initialized, it takes me straight to the "Your project has no data" page. Shouldn't we be showing a message saying you need to initialize DVC?

Reply:
Ah, wait, I'm seeing this behavior on main as well.

Reply:
😱

Reply:
@julieg18 I cannot recreate this:

https://user-images.githubusercontent.com/37993418/209230803-d7414ba9-6928-4f2f-8575-9be905d56438.mov

Are you sure the workspace that you were testing against didn't contain a DVC project?

Reply:
> @julieg18 I cannot recreate this:

Probably an issue on my end. 

> Are you sure the workspace that you were testing against didn't contain a DVC project?

Yes, pretty sure. It shows no file when I try `find . -name .dvc`. I'm getting this error when the extension or I try running a dvc command:

```
ERROR: /Users/juliegalvan is not a git repository
```

Reply:
that scenario is covered by #2947

Reply:
for anyone that arrives here: debugged on a call with @julieg18 

there was a `.dvc` folder under `/Users/juliegalvan` which was causing some very unexpected/confusing behaviour. After deleting that folder everything was resolved. An issue will be raised in the DVC repo to discuss

Reply:
Issue created! https://github.com/iterative/dvc/issues/8731